src/archana/SeleniumAss3.java

```diff
@@ -0,0 +1,51 @@
+package archana;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class SeleniumAss3 {
+	static WebDriver driver;
+
+	void launchBrowser(String url) {
+		String os = System.getProperty("os.name").toLowerCase();
+		System.out.println("os : " + os);
+		String path = os.contains("windows") ? "./resources/windows/chromedriver.exe"
+				: os.contains("mac") ? "./resources/mac/chromedriver" : null;
+		System.setProperty("webdriver.chrome.driver", path);
+		driver = new ChromeDriver();
+		driver.get(url);
+	}
+
+	void faceBookSignUp() {
+		launchBrowser("https://www.facebook.com/r.php");
+		// facebook page title verification
+		if (driver.getTitle().equals("Sign up for Facebook | Facebook"))
+			System.out.println("Pass--Facebook Signup page loaded sucessfully");
+		else
+			System.out.println("Fail--Facebook Signup page not loaded-Incorrect page ");
+		// enter Details
+		driver.findElement(By.xpath("//input[@name=\"firstname\"]")).sendKeys("Archa");
```

Comment:
Do you Observe why there is '\' is coming ? 
Please always use a single inverted comma for the attribute value.